conf/README.md

```diff
@@ -41,21 +38,172 @@ All of the below sections, will be available from the ocsci config dataclass.
 Framework RUN related config parameters. If the parameter is for the complete
 run it belongs here.
 
+* `username` - Kubeadmin username
+* `password_location` - Filepath (under the cluster path) where the kubeadmin password is located
+* `log_dir` - Directory where logs are placed
+* `run_id` - Timestamp ID that is used for log directory naming
+* `kubeconfig_location` - Filepath (under the cluster path) where the kubeconfig is located
+* `cli_params` - Dict that holds onto all CLI parameters
+* `client_version` - OCP client version
+* `bin_dir` - Directory where binaries are downloaded to
+* `google_api_secret` - Filepath to google api secret json file
+* `force_chrome_branch_base` - Chrome base branch for openshift console UI testing
+* `force_chrome_branch_sha256sum` - Chrome branch sha256sum for openshift console UI testing
+* `chrome_binary_path` - Filepath to the chrome browser binary
+* `io_in_bg` - Run IO in background (Default: false)
+* `io_load` - Target percentage for IO in background
+* `log_utilization` - Enable logging of cluster utilization metrics every 10 seconds. Set via --log-cluster-utilization
+* `use_ocs_worker_for_scale` - Use OCS workers for scale testing (Default: false)
+* `load_status` - Current status of IO load
+
 #### DEPLOYMENT
 
 Deployment related parameters. Only deployment related params not used
 anywhere else.
 
+* `installer_version` - OCS installer version
+* `force_download_installer` - Download the OCS installer even if one already exists in the bin_dir
+* `force_download_client` - Download the OCP client even if one already exists in the bin_dir
+* `default_latest_tag` - OCS latest tag to be used by default if one is not provided
```

Comment:
do you think we need to explicitly mentions it as "OCS latest image tag" ?

Reply:
I noticed a few different config settings that should probably be renamed to be more explicit. I'd like to tackle those in a separate PR since it will require actual code changes. This can definitely be one of the settings we refactor.